runtime/ftplugin/mom.vim

```diff
@@ -0,0 +1,13 @@
+" Vim filetype plugin
+" Language:	roff(7)
+" Maintainer:	Aman Verma
```

Comment:
Again, you are adding a new filetype, but setting someone else as maintainer. That doesn't sound right.

Reply:
It might not, but again these files used to be detected as NROFF files, so it makes sense to keep them maintained as before for the time being instead of hijacking. Mind that this is just a request, though both ftplugin and syntax maintainers were notified and expressed interest, and can linger untill all responsibilities are settled. I have little knowledge of NROFF and MOM, but thought MOM as a light-weight LaTeX alternative interesting enough to detect it at least and start from there